### PR TITLE
googlemaps - Added remove method to MapsEventListener class

### PR DESCRIPTION
**Pull request by SimonPedley**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test googlemaps`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.com/maps/documentation/javascript/reference/event#MapsEventListener
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by SimonPedley** [comment]
Hi!
This is my first open source contribution. A tiny change just to ease me into it and see how it all works.
Thanks!

File: types/googlemaps/index.d.ts [hunk 1]
```diff
@@ -304,7 +304,9 @@ declare namespace google.maps {
             stop(): void;
         }
 
-        // TODO find source documentation
-        class MapsEventListener {}
+        // https://developers.google.com/maps/documentation/javascript/reference/event#MapsEventListener
+        class MapsEventListener {
+            remove(): void;
```
Comment (demensky) [inline types/googlemaps/index.d.ts:309]: The [remove](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/googlemaps/reference/event.d.ts#L71) method has long been present in `google.maps.MapsEventListener`.

Comment (demensky) [reply types/googlemaps/index.d.ts:309]: `class MapsEventListener` remains as a rudiment and will be removed in the future.

Comment (SimonPedley) [reply types/googlemaps/index.d.ts:309]: Ah of course, i did think it was a bit odd that this was missing.
Appologies. I didn't realise there were duplicate type definitions of the MapsEventListener class and had managed to find my self using this one.

Thanks for your help. I'll close this.

**Pull request closed without merging**